app/upgrades/v12/upgrades.go

```diff
@@ -16,6 +16,8 @@ func CreateV12UpgradeHandler(
 	keepers *keepers.AppKeepers,
 ) upgradetypes.UpgradeHandler {
 	return func(ctx sdk.Context, _ upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+		// transfer module consensus version has been bumped to 2
+		// the above is https://github.com/cosmos/ibc-go/blob/v5.1.0/docs/migrations/v3-to-v4.md
```

Comment:
missing to add  `ibcfeetypes.ModuleName` ?
https://github.com/CosmosContracts/juno/blob/f9ae0c654e2c034cc1c71701f191421dd94b8b88/app/upgrades/v12/constants.go#L17

Reply:
Ahh yes thank you. I was having consensus version issues before but forgot to write the patch back in